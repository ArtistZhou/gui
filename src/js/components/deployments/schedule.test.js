import React from 'react';
import renderer from 'react-test-renderer';
import Schedule from './schedule';

it('renders correctly', () => {
  const tree = renderer.create(<Schedule schedule={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
