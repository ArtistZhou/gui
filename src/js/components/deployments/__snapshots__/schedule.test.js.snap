// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h3>
    Scheduled deployments
  </h3>
  <div
    className={null}
  >
    <p
      className="italic"
    >
      No deployments scheduled
    </p>
  </div>
</div>
`;
